src/parachain/pools.ts

```diff
@@ -13,46 +13,47 @@ import {
     newMonetaryAmount,
     tickerToCurrencyIdLiteral,
 } from "../";
+import { WrappedCurrency } from "../types";
 import { newAccountId } from "../utils";
 
 export interface PoolsAPI {
```

Comment:
I think this should be called `RewardsAPI` to account for the staking/rewards pallet name.